README.rst

```diff
@@ -38,31 +49,37 @@ Our Team
 
 PyMedPhys is what it is today due to its maintainers and developers. The
 currently active developers and maintainers of PyMedPhys are given below
-in alphabetical order along with their affiliation:
```

Comment:
Matt asked that I remove the alphabetical idea. We'll see how we go with this list into the future.

Reply:
No worries